manifests/init.pp

```diff
@@ -173,8 +173,8 @@
   Enum['on', 'off'] $ssl_stapling_verify                     = 'off',
   Stdlib::Absolutepath $snippets_dir                         = $nginx::params::snippets_dir,
   Boolean $manage_snippets_dir                               = true,
-  $types_hash_bucket_size                                    = '512',
-  $types_hash_max_size                                       = '1024',
+  Variant[Integer,String] $types_hash_bucket_size            = '512',
+  Variant[Integer,String] $types_hash_max_size               = '1024',
```

Comment:
Correct to allow strings, (ie `4k` is an allowable value). I wonder if an `Nginx::Size` type alias would be useful?
http://nginx.org/en/docs/syntax.html

Reply:
and I guess also `Nginx::Time_interval`??

Reply:
Looks like `Nginx::Size` and `Nginx::Time` already exist